packages/rs-dpp/src/errors/consensus/basic/basic_error.rs

```diff
@@ -72,6 +74,15 @@ pub enum BasicError {
     #[error(transparent)]
     VersionError(VersionError),
 
+    #[error(transparent)]
+    ContractError(DataContractError),
+
+    #[error(transparent)]
+    UnknownSecurityLevelError(UnknownSecurityLevelError),
+
+    #[error(transparent)]
+    UnknownStorageKeyRequirementsError(UnknownStorageKeyRequirementsError),
+
```

Comment:
Are those contract errors or common errors for multiple types?

Reply:
I think they are identity key errors.